tests/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 
 f = os.readlink(__file__) if os.path.islink(__file__) else __file__
-path = os.path.realpath(os.path.join(f, '..', '..'))
+path = os.path.realpath(os.path.join(f, '..', '..', 'src'))
```

Comment:
This entire file should be empty. It is inappropriate to modify `sys.path` as part of the tests.

As with the integration tests, we should be testing *as installed*.